examples/with-solana/README.md

```diff
@@ -166,7 +166,138 @@ Token balance for warchest: 0.0001
+Turnkey Solana wallet address: 2bwUkGZ72SNyzBWFwfzKVrVWJ9JNZggjqFPDM9X9mrp9
+Token Mint public key: FA7umztm6eYcENE22ndEZhd8MCf9C7myJ2KjfCwobj8F
+Non root user created with user id: 13c4609b-8818-40be-aeac-c8b63da0de4a
+```
+
+#### Step 2. Attempt to Transfer Tokens (This is expected to FAIL)
+
+We will now attempt to transfer these SPL tokens that we've created
+
+```bash
+$ pnpm allow-token-transfer attempt_transfer
+```
+
+This will prompt you first to enter in the Solana wallet address that is originating the transfer (use the address labeled "Turnkey Solana wallet address" output at the end of the setup step)
+
+```bash
+? Enter Solana wallet address originating transfer (created during setup stage):
+```
+
+Once you've entered your wallet address you will then be prompted to enter in the Mint address/public key for the token being transferred (use the address labeled "Token Mint public key" output at the end of the setup step)
+
+```bash
+? Enter Mint account address of token being transferred (created during setup stage):
+```
+
+Since this is being initiated by a non root user, and the policy allowing this has not been created, this is EXPECTED TO FAIL. You will get an error that includes the following information saying that no policies evaluated to the outcome: Allow
+
+```bash
+details: [
+    {
+      '@type': 'type.googleapis.com/errors.v1.PolicyEnginePermissionError',
+      message: 'No policies evaluated to outcome: Allow',
+      policyEvaluations: []
+    }
+  ],
+  code: 7
+```
+
+#### Step 3. Create Policy allowing the correct user to make SPL transfers to receiving Associated Token Address
+
+This is the most important step! In this step we are taking our receiving address (the Turnkey Warchest), calculating it's "associated token address", given the Mint account address for the token we've created, and creating a policy that allows the correct non root user to sign Solana transactions if and only if they contain a single instruction that is an SPL token transfer to this associated token address.
+
+```bash
+$ pnpm allow-token-transfer create_token_policy
+```
+
+This will prompt you first to enter in the user ID of the non root user who's credentials are being used to sign the transfer (use the value labeled "Non root user created with user id" output at the end of the setup step)
+
+```bash
+? Enter non-root user ID originating the transfer (created during setup stage):
+```
+
+Once you've entered this user id, you will then be prompted to enter in the Mint address/public key for the token being transferred (use the address labeled "Token Mint public key" output at the end of the setup step)
+
+```bash
+? Enter Mint account address of token being transferred (created during setup stage):
+```
+
+This should successfully create your policy!
+
+```bash
+New policy created!
+- Name: Let non root user send SPL transfers to the ATA of WARCHEST: 6nDvKk6emwskFLtEpbQgFX6rsMtzbNY4LkvaNnzkvBaq
```

Comment:
should we specify which SPL token we're working with given ATA's are token-specific?

Reply:
did you mean to resolve this one? you could also mention that we're allowing specifically the transfer to the ATA of the warchest wallet associated with the newly created token